playwright/src/main/java/com/microsoft/playwright/junit/Config.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.APIRequest;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Config {
+  private BrowserType.LaunchOptions launchOptions = new BrowserType.LaunchOptions();
+  private Browser.NewContextOptions contextOption = new Browser.NewContextOptions();
+  private APIRequest.NewContextOptions apiRequestOptions = new APIRequest.NewContextOptions();
+  private Map<String, Object> clientValues = new HashMap<>();
+
+  private String browserName = "chromium";
+
+  public String browserName() {
+    return browserName;
+  }
+
+  public void setBrowserName(String browserName) {
+    this.browserName = browserName;
+  }
+
+  public void setHeadless(boolean headless) {
+    launchOptions.setHeadless(headless);
+  }
+
+  public void setBaseURL(String url) {
+    contextOption.setBaseURL(url);
+  }
+
+  public void setStorageStatePath(Path path) {
+    contextOption.setStorageStatePath(path);
+  }
+
+  public BrowserType.LaunchOptions launchOptions() {
+    return launchOptions;
+  }
+
+  public Browser.NewContextOptions contextOptions() {
+    return contextOption;
+  }
+
+  public APIRequest.NewContextOptions apiRequestOptions() {
+    return apiRequestOptions;
+  }
+
+  public <T> void setClientValue(String name, T value) {
```

Comment:
is this for setting PlaywrightOptions?

Reply:
The idea here was to have a storage for the clients to pass some options to their tests. I.e. make the Config available in the tests (e.g. as a method parameter similar to Browser etc) and if the client needs to pass some custom properties they could achieve that by using these client property map.

Reply:
We can make the whole Config object available to the test so I'm not sure if we need this but if you want it I can add it.

Reply:
Let's omit for now. Even if we expose Config object it won't have a getter setter for `myCustomProperty` and this is essentially a custom hash map on the config.